docs/metrics/README.md

```diff
@@ -356,18 +356,13 @@ predictable and reliable behavior when excessive cardinality happens, whether it
 was due to a malicious attack or developer making mistakes while writing code.
 
 OpenTelemetry has a default cardinality limit of `2000` per metric. This limit
-can be configured at `MeterProvider` level using
+can be configured at `MeterProvider` level using the
 `SetMaxMetricPointsPerMetricStream` method, or at individual
 [view](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view)
-level. Refer to this
+level using `MetricStreamConfiguration.CardinalityLimit`. Refer to this
 [doc](../../docs/metrics/customizing-the-sdk/README.md#changing-maximum-metricpoints-per-metricstream)
```

Comment:
This section can be updated to be "changing-the-cardinalityLimit-of-a-metric" given the name standardization effort: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5328/files.